File: program/convert-paise-to-rupees/ConvertPaiseToRupees.java
import java.util.Scanner;

class ConvertPaiseToRupees {
	
	public static void main(String[] args) {
		
		Scanner input = new Scanner(System.in);
		
		System.out.print("Input ( Paise ) : ");
		int currencyInPaise = input.nextInt();
		
		System.out.print("Output ( Rupees ) : " + (currencyInPaise / 100));
	}
	
	
}

File: program/program/convert-paise-to-rupees/ConvertPaiseToRupees.kt
fun main() {
    print("Enter paise : ")
    println()
    var input = readLine()

    println("$input pasie = " + ConvertPaiseToRupees(input!!.toDouble()) + " rupees")

}

private fun ConvertPaiseToRupees(paise: Double): Double {
    var rupees = paise / 100

    return rupees
}
